### PR TITLE
00836 d move basic tests out of jrs

### LINKED ISSUE #836: Move light TPS JRS regression test flow to pure CircleCi test workflow.
**Summary**
Right, we have lots of test workflows in JRS framework. Some of them are just basic functional tests and have very low TPS rate. Running these regression tests in JRS regression is probably not the best use of these resources and also waste of money. 

So it would be better to run the light TPS tests in CircleCi controlled workflow with spot instances.

**Possible resolution**
1. Identify these regressions first. Right now, the AWS-Daily-Services-Comp-Basic-4N-1C's **Misc-Basic-1-11M** is one of them;
2. Either create new worklfows in CircleCi or merge them into existing CircleCi workflow;

### PR DESCRIPTION
**Pull request by ljianghedera**
**Related issue(s)**:
Closes #836 

**Summary of the change**:
1. Disabled the daily basic JRS regression test. Didn't delete it because we may bring it back after some thinking;
2. Created a CircleCi regression test workflow to cover the JRS regression test disabled;
3. Modified the superuser throttling test case to not run into INSUFFICIENT_ACCOUNT_BALANCE error;

The successful test run: https://app.circleci.com/pipelines/github/hashgraph/hedera-services/7546/workflows/74da5c55-0a3f-4a64-b13a-e366e660b998

**External impacts**:
None.

**Applicable documentation**
- [ ] Javadoc
- [ ] HAPI protobuf comments
- [ ] README

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -891,7 +891,7 @@ workflows:
           filters:
             branches:
               only:
-                - master
+                - Disable-X-master
```
Comment (qnswirlds) [inline .circleci/config.yml:894]: What workflow is this? Should this change be reverted back?

File: .circleci/config.yml [hunk 2]
```diff
@@ -1354,6 +1354,44 @@ workflows:
...
+      - schedule:
+          cron: "15 4 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - fast-build-artifact:
+          context: Slack
+          workflow-name: freeze restart
```
Comment (qnswirlds) [inline .circleci/config.yml:1368]: copy-paste issue I think. `workflow-name` should be `nightly-comprehensive-functional-regression` I guess

File: .circleci/config.yml [hunk 3]
```diff
@@ -1354,6 +1354,44 @@ workflows:
...
+      - start-singlejob-testnet:
+          context: Slack
+          requires:
+            - fast-build-artifact
+          workflow-name: "nightly-comprehensive-functional-regression"
+          infra-branch: services-with-platform070
```
Comment (qnswirlds) [inline .circleci/config.yml:1374]: Why do we need this infra branch? Shouldn't it be the default `hashgraph-hedera-services`?

**Review by qnswirlds — COMMENTED** [review]
And my comment in the issue https://github.com/hashgraph/hedera-services/issues/836#issuecomment-738459828

File: .circleci/config.yml (hunk 1 above)
Comment (ljianghedera) [reply .circleci/config.yml]: No, this workflow is not needed here per discussion with Nosh.

File: .circleci/config.yml (hunk 2 above)
Comment (ljianghedera) [reply .circleci/config.yml]: Yeah, fixed.

File: .circleci/config.yml (hunk 3 above)
Comment (ljianghedera) [reply .circleci/config.yml]: Done, the default  is used. Thanks for the good catch.

**Review by qnswirlds — APPROVED** [review]
LGTM. My comment still holds, though https://github.com/hashgraph/hedera-services/issues/836#issuecomment-738459828

**Pull request merged**